frontend/components/elements/StoryItem.vue

```diff
@@ -124,42 +123,34 @@
                 </BDropdown>
             </td>
 
-            <td v-if="isView('backlog')">
+            <td v-if="isView(['backlog', 'planning-backlog', 'planning-sprint'])">
                 <BButton 
                     size="sm" 
-                    variant="primary" 
-                    title="Edit" 
+                    variant="outline-primary"
+                    title="Edit"
```

Comment:
The edit button should have the same size as the delete button.

Reply:
they're the same size for me? unless you mean the width, which is dependent on the width of the icon + text, and I can't really make the word edit longer...

Reply:
You could specify the width to about 75px for each button of them:
![image](https://user-images.githubusercontent.com/7222193/34920204-40eb1602-f96f-11e7-8369-56e0d9d98d62.png)


It looks much more cleaner.

Reply:
I actually think the opposite - seeing as icons and text are not aligned anymore, it looks a lot messier to me. if we'd make the buttons fixed width, they should all be at least a bit bigger than the widest one (the delete one in this case), with the icon aligned left and the text center in the remaining space, to create a unifying look - however seeing as space in that table is already limited, I would not want to make the delete button any wider, especially seeing as we don't win anything imo - if we're focusing on this, there's a lot more issues with that page we should look at first...

Reply:
ok :)